### PR TITLE
make --gcp_region_id optional

### PR DESCRIPTION
**Pull request by thinhha**

### CODE REVIEW

File: clouddq/main.py [hunk 1]
```diff
@@ -70,15 +70,17 @@
 @click.option(
     "--gcp_region_id",
     help="GCP region used for running BigQuery Jobs and for storing "
-    " any intemediate DQ summary results. "
+    " any intemediate DQ summary results. This is an optional argument. "
```
Comment (pbalm) [inline clouddq/main.py:73]: intermediate (typo)

File: clouddq/main.py [hunk 2]
```diff
@@ -276,15 +276,12 @@ def main(  # noqa: C901
...
-        )
-    ) or (dbt_profiles_dir and (gcp_project_id or gcp_bq_dataset_id or gcp_region_id)):
+    if (not dbt_profiles_dir and (not gcp_project_id or not gcp_bq_dataset_id)) or (
+        dbt_profiles_dir and (gcp_project_id or gcp_bq_dataset_id)
+    ):
         raise ValueError(
-            "CLI input must define connection configs using '--dbt_profiles_dir' or "
-            "using '--gcp_project_id', '--gcp_bq_dataset_id', and '--gcp_region_id'."
+            "CLI input must define connection configs using either '--dbt_profiles_dir' "
+            "or ('--gcp_project_id', '--gcp_bq_dataset_id', '--gcp_region_id')."
```
Comment (pbalm) [inline clouddq/main.py:284]: I'm not sure the notation is going to be understood by everyone. Maybe put "...or one of: --gcp_project_id, ..."

**Comment by pbalm** [comment]
Two minor textual things but otherwise: LGTM

**Review by pbalm — DISMISSED** [review]

**Review by AmandeepSinghCS — APPROVED** [review]
LGTM. Thanks

File: clouddq/main.py (hunk 1 above)
Comment (thinhha) [reply clouddq/main.py:73]: ack & fixed

File: clouddq/main.py (hunk 2 above)
Comment (thinhha) [reply clouddq/main.py:284]: ack & fixed

**Review by pbalm — APPROVED** [review]
LGTM

**Pull request merged**